### DISCUSSION TITLE
Changing osqp options in python with opti stack and 'sqpmethod'

### DISCUSSION THREAD

**Discussion by davidhnguyen2000** [casadi/casadi#3541] [Q&A] [answered]
I am trying to alter the OSQP solver options in python using `opti.solver('sqpmethod', s_opts)` where I have my osqp options as: `qp_opts = {"max_iter":20}` and my sqpmethod options as: `s_opts = {"qpsol": 'osqp', "max_iter": 20, "hess_lag": hessLag, "error_on_fail": False, "osqp": qp_opts}` according to the documentation. 

But when running this I get the following error saying that osqp is an unknown option: `RuntimeError: Error in Opti::to_function [OptiNode] at .../casadi/core/optistack.cpp:331:
Error in Opti::to_function [OptiNode] at .../casadi/core/optistack.cpp:331:
.../casadi/core/options.cpp:244: Unknown option: osqp`. Is there a different way to set the osqp options? If I use "qpsol_options", "max_iter" is also not an option valid for qpsol_options. Any help is much apprecieated thank you!

**Comment by jgillis** [comment, marked as answer]
Almost there: qpsol_options -> qpsol -> max_iter
